Makefile

```diff
@@ -397,6 +397,9 @@ test: $(LINUXKIT) test-images-patches | $(DIST)
 	cp pkg/pillar/results.xml $(DIST)/
 	$(QUIET): $@: Succeeded
 
+test-profiling:
+	make -C pkg/pillar test-profiling
+
```

Comment:
Should this be documented like the other Makefile targets (in "make all" output)?

Reply:
Yes, I added it to `make help`